### ISSUE TITLE
E262 separation of "too many" "not enough"

### ISSUE THREAD

**Issue by beauxq** [astral-sh/ruff#16766]
### Summary

E262 is reporting both:
- cases where there are fewer than 1 spaces after the `#`
  - `#comment`
- cases where there are more than 1 spaces after the `#`
  - `#   comment`

I would like a report when there are fewer than 1 spaces after the `#` but no report when there are more than 1 spaces.

Can this be configurable? or separated into different rules?

**Comment by MichaReiser** [comment]
Can you tell me more about why you want to allow more than one space?

I don't think it makes sense to make this rule configurable because the goal of the pycodestyle rules is to reflect the recommendations from PEP8 and that is:

> Per [PEP 8](https://peps.python.org/pep-0008/#comments), inline comments should start with a # and a single space.

**Comment by beauxq** [comment]
The pattern in the code that I was trying to bring ruff into:
```python
    {
        a: b,  #   1 -  25
        c: d,  #  26 -  50
        e: f,  #  51 -  75
        g: h,  #  76 - 100
        i: j,  # 101 - 125
    }
```
PEP8 even has an exception for the single space in block comments:
> Each line of a block comment starts with a # and a single space (unless it is indented text inside the comment).

The way I see it, a single comment could be both an inline comment and a block comment at the same time. That's what this example shows.

**Comment by MichaReiser** [comment]
Thanks for sharing. I can see how this formatting is desirable but it's out of this rule's scope (which is PEP8 centered).
